content/_data/upgrades/2-303-1.adoc

```diff
@@ -0,0 +1,176 @@
+As Jenkins has developed further, capabilities moved from inside Jenkins core to dedicated Jenkins plugins, like the plugin:external-monitor-job[External Monitor Job plugin], the plugin:ldap[LDAP plugin], and the plugin:pam[PAM authentication plugin].
+When those plugins were created, the plugins were "bundled" inside the jenkins.war file to retain compatibility for plugins that depended on their functionality to be inside Jenkins core.
+The plugin:external-monitor-job[External Monitor Job plugin], the plugin:ldap[LDAP plugin], and the plugin:pam[PAM authentication plugin] are no longer bundled with Jenkins.
+
+In very rare cases, this could result in problems when attempting to install plugins compatible with Jenkins (then Hudson) before 1.467.
+If you use a plugin that relies on the functionality provided by the plugin:external-monitor-job[External Monitor Job plugin], the plugin:ldap[LDAP plugin], or the plugin:pam[PAM authentication plugin] and manage plugins outside the Jenkins plugin manager, you will now need to ensure yourself that a recent release of those plugins are installed.
+Jenkins will attempt to load such plugins but may fail at any time during startup or afterwards with `ClassNotFoundException` or similar.
+In these cases, the best path forward is usually to modify the plugin that has the problem so that it requires a newer version of Jenkins core.
+
+==== Removed JEP-200 compatibility workarounds
+
+JEP-200 workarounds for several plugins have been removed.
+If you are using any of the following plugins, make sure you use at least the versions specified:
+
+* Maven Integration 3.1 (released Jan 2018)
+* Job DSL 1.67 (released Jan 2018)
+* Monitoring 1.71.0 (released Feb 2018)
+* Git Client 2.7.1 (released Jan 2018)
+* Pipeline: Supporting APIs 2.17 (released Jan 2018)
+* OWASP Dependency-Check 3.1.1 (released Jan 2018)
+
+==== Removed `jna-posix` from Jenkins core
+
+The `jna-posix` dependency has been removed from Jenkins core.
+Plugins that use `jna-posix` functionality, including
+plugin:maven-repo-cleaner[Maven Repository Scheduled Cleanup],
+plugin:sicci_for_xcode[SICCI for Xcode], and
+plugin:tmpcleaner[java.io.tmpdir cleaner]
+are no longer supported and must be removed.
+
+==== Removed `JTidy` from Jenkins core
+
+The `JTidy` dependency has been removed from Jenkins core.
+Users of the plugin:jdepend[JDepend plugin] must upgrade to the latest version.
+Other plugins that use JTidy functionality, including plugin:nis-notification-lamp[NIS notification lamp], must be updated to explicitly declare a dependency on JTidy rather than relying on Jenkins core to provide this library.
+
+==== Removed Bytecode Compatibility Transformer from Jenkins core
+
+Support for plugins that rely on the `hudson.model.Queue$Item#id` or `hudson.model.AbstractProject#triggers fields`, including plugin:slave-prerequisites[Slave Prerequisites plugin] and plugin:vertx[vertx plugin], has been dropped.
```

Comment:
Not sure we want to have "slave" here.
Also reference the library?

Reply:
I'm not sure what else to use to describe it except "slave prerequisites plugin".  It was last released 9 years ago, long before the decision to use more inclusive terminology.  Any recommendations for replacement text?